airflow/dag/serialization/serialization.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Utils for DAG serialization with JSON."""
+
+import datetime
+import json
+import logging
+from typing import Dict, Optional, TYPE_CHECKING, Union
+
+import dateutil.parser
+import jsonschema
```

Comment:
This is a new direct dep on Airflow, and we should add it to our setup.py

Reply:
Added in https://github.com/apache/airflow/pull/5743